kura/test/org.eclipse.kura.core.configuration.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,24 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: org.eclipse.kura.core.configuration.test
+Bundle-SymbolicName: org.eclipse.kura.core.configuration.test;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Bundle-Vendor: Eclipse Kura
+Bundle-RequiredExecutionEnvironment: JavaSE-1.6
+Import-Package: org.eclipse.kura;version="[1.2,2.0)",
+ org.eclipse.kura.configuration;version="[1.1,2.0)",
+ org.eclipse.kura.configuration.metatype;version="[1.0,2.0)",
+ org.eclipse.kura.core.configuration;version="1.0.2",
```

Comment:
I guess it would be better to make this test a fragment of the org.eclipse.kura.core.configuration plugin (see #832) avoiding to import org.eclipse.kura.core.configuration.*